.github/ISSUE_TEMPLATE/launch-security.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Launch - Security
+about: Prepare for a site launch with a checklist of common steps.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
+
+- [ ] Consider installing & configuring the SecKit module.
+- [ ] Verify that development related accounts/passwords have been reset or removed.
+- [ ] Verify that proper permissions are set on settings.php and services.yml files.
+- [ ] Confirm that development modules are disabled on production (e.g. kint, devel_generate, devel, etc).
+- [ ] Verify that `$settings['trusted_host_patterns']` is configured.
+- [ ] Verify that cron runs from page requests are disabled (`$settings['update_free_access'] = FALSE;)`.
```

Comment:
Is this a security concern? I’ve always thought of it as a performance concern, but I could be missing something. Maybe it’s both?

Reply:
From the `default.settings.php` file:
```
/**
 * Access control for update.php script.
 *
 * If you are updating your Drupal installation using the update.php script but
 * are not logged in using either an account with the "Administer software
 * updates" permission or the site maintenance account (the account that was
 * created during installation), you will need to modify the access check
 * statement below. Change the FALSE to a TRUE to disable the access check.
 * After finishing the upgrade, be sure to open this file again and change the
 * TRUE back to a FALSE!
 */
$settings['update_free_access'] = FALSE;
```

My item was all wrong though. It was mixing cron and update concerns. I separated these out in a new commit.

4176e806123a7a8fc79551621b176502a6d56a9b